src/components/Common/TitleAndMediaList.js

```diff
@@ -59,7 +60,7 @@ const TitleAndMediaList = ({
       </MediaItems>
       {CTALink && CTAText ? (
         <Padding top={3}>
-          <StyledLink external href={CTALink} title={CTAText}>
+          <StyledLink external={external} href={CTALink} title={CTAText}>
```

Comment:
We need to make sure that we can go to either a local page with `to` prop or an external link with `href` prop.

You're setting `external` to false but not providing a `to` prop so you don't end up with a Gatsby `Link` component, just an anchor with an href which will cause issues with how gatsby routing works.

Remember how StyledLink works:
```
if (to) {
    return (
      <Anchor {...externalProps} to={to} {...props}>
        {children}
      </Anchor>
    )
  }

  return (
    <Anchor {...externalProps} as="a" href={href} {...props}>
      {children}
    </Anchor>
  )
```

```
const styledLinkProps = {
    title: CTAText,
    ...(external ? { external: true, href: CTALink }  : { to: CTALink}),
}

<StyledLink {...styledLinkProps}>CTAText</StyledLink>
```